scripts/concurrent_task_utils_test.py

```diff
@@ -45,7 +45,7 @@ def mock_print(*args):
             linter_stdout list.
 
             Args:
-                *args(str): Variable length argument list of values to print in
+                *args: str. Variable length argument list of values to print in
```

Comment:
The comment says `Variable length argument list` and type says `str`. Also, we have different type for `*args` everywhere - shouldn't there be a single type everywhere and shouldn't we have a check for that?

Reply:
Great, Thanks!

Reply:
I've included it in this check.

Reply:
Ok, great!